docs: Add 2023-04-06 meeting notes (#422)
eslint/tsc-meetings: notes/2023/2023-04-06.md

### Time allocation of Core Contributors

**TSC Summary:** Because we are down to two members on the TSC and primary contributors to ESLint, we may need to adjust how we are spending time. In particular, core contributors should be spending more time on writing code and less time on other stuff. Especially now that Sam and Joel are helping with triage and support, it seems like a good time for core members to step back a bit from that and try to make more progress on a roadmap.

**Resolution:** We've agreed to spend more time coding and targeting at least 25% of our time for that purpose.

**Action Item:** @nzakas will continue to break down the larger tasks discussed into checklists to hopefully make it easier for us to pick off smaller pieces that are easier to finish off

### Require Minimal Reproducible Demo

**TSC Summary:** Do we want to add a requirement on bug reports for a minimal reproduction case with either:

1. Playground link
2. Stackblitz link
3. GitHub repo?

Some other projects are requiring this and it seems like it could further reduce time to evaluate and close these issues. For issues related to rules, playground links are really easy to create.

> **Resolution:** we've agreed to add a field to require a repro link in bug reports. I'll take the action item to update the template.

Reviewer: Just need to call out the action item.

```suggestion
**Resolution:** we've agreed to add a field to require a repro link in bug reports.

**Action Item:** @nzakas to update the issue template
```

# 2023-04-06 ESLint TSC Meeting Notes

## Transcript

[`2023-04-06-transcript.md`](2023-04-06-transcript.md)
## Attending

* Nicholas C. Zakas (@nzakas) - TSC
* Milos Djermanovic (@mdjermanovic) - TSC

@nzakas moderated, and @sam3k took notes.

## Topics

> ### [Commits should create a release pull request](https://github.com/eslint/create-config/pull/56)

Reviewer 2: ```suggestion
### Commits should create a release pull request
```

This topic wasn't related to the linked PR.

**TSC Summary:**  We've decided to configure release-please so that all commits will trigger the creation of a release pull request. The action item is for .

**Action Item:** @mdjermanovic will submit a pull request to modify this behavior

### Encourage transition to FlatConfig

> @nzakas:
>
> Now that flat config is feature complete, do we want to announce that and encourage people to start transitioning their configs?

> **Resolution:** We've decided to announce flat config as feature complete. The action item is for me to write a blog post announcing it.

Reviewer 2: ```suggestion
**Resolution:** We've decided to announce flat config as feature complete.

**Action Item:** @nzakas will write a blog post announcing it.
```
